fluentgen/src/test/java/com/azure/autorest/fluent/model/clientmodel/fluentmodel/DeduplicateSetterTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+
+package com.azure.autorest.fluent.model.clientmodel.fluentmodel;
```

Comment:
the test is on template, should it in fluent.template package?

Reply:
Ok.

Reply:
Changed it to template package and renamed it `ModelTemplateTests`.